extensions/wikia/Email/Controller/CommentController.class.php

```diff
@@ -78,17 +78,17 @@ public function body() {
 			'editorUserName' => $this->getCurrentUserName(),
 			'editorAvatarURL' => $this->getCurrentAvatarURL(),
 			'summary' => $this->getSummary(),
-			'details' => $this->getDetails(),
```

Comment:
Ok, it took me a long time to figure out *this* was the change that addressed the bug for SOC-812.  It seems that calling ```getDetails``` has a side effect that screws up creating the link in ```getCommentSectionLink```.  My recommendation would be:

1) Document in a comment that the 'details' line is position dependent so others know not to touch it
2) Realize this is a terrible state of things
3) Erase the comment, move the line back up and find the root cause of the problem ;)

I found the issue, details below...